pkg/operator/sync.go

```diff
@@ -62,6 +72,97 @@ func (optr *Operator) syncAll(rconfig *renderConfig, syncFuncs []syncFunc) error
 	return agg
 }
 
+func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
+	if err := optr.syncCustomResourceDefinitions(); err != nil {
+		return err
+	}
+
+	if optr.inClusterBringup {
+		glog.V(4).Info("Starting inClusterBringup informers cache sync")
+		// sync now our own informers after having installed the CRDs
+		if !cache.WaitForCacheSync(optr.stopCh, optr.mcpListerSynced, optr.mcListerSynced, optr.ccListerSynced) {
+			return errors.New("failed to sync caches for informers")
+		}
+		glog.V(4).Info("Finished inClusterBringup informers cache sync")
+	}
+
+	// sync up the images used by operands.
+	imgsRaw, err := ioutil.ReadFile(optr.imagesFile)
+	if err != nil {
+		return err
+	}
+	imgs := Images{}
+	if err := json.Unmarshal(imgsRaw, &imgs); err != nil {
+		return err
+	}
+
+	// sync up CAs
+	etcdCA, err := optr.getCAsFromConfigMap("openshift-config", "etcd-serving-ca", "ca-bundle.crt")
+	if err != nil {
+		return err
+	}
+	etcdMetricCA, err := optr.getCAsFromConfigMap("openshift-config", "etcd-metric-serving-ca", "ca-bundle.crt")
+	if err != nil {
+		return err
+	}
+	rootCA, err := optr.getCAsFromConfigMap("kube-system", "root-ca", "ca.crt")
+	if err != nil {
+		return err
+	}
+	// as described by the name this is essentially static, but it no worse than what was here before.  Since changes disrupt workloads
```

Comment:
i know that this comment predates this PR but I dont quite understand what it is trying to say?

Reply:
worth asking @deads2k maybe